### PR TITLE
fix: remove node globals

### PR DESCRIPTION
**Pull request by hugomrdias**
This PR remove node globals, updates deps and fixes lint problems.

related to this https://github.com/ipfs/js-ipfs/issues/2924

### CODE REVIEW

File: test/garbage.spec.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ const garbage = require('garbage')
 
 const cbor = require('../src')
 
-const CBOR_GARBAGE = process.env.NODE_CBOR_GARBAGE
-const NO_GARBAGE = process.env.NO_GARBAGE
-const REPEATS = parseInt(CBOR_GARBAGE || 1000)
+// const CBOR_GARBAGE = global.process.env.NODE_CBOR_GARBAGE
```
Comment (vmx) [inline test/garbage.spec.js]: Why not just removing those instead of commenting out?

Comment (hugomrdias) [reply test/garbage.spec.js]: just wanted to confirm with the reviewer if this would be better removed

Comment (dignifiedquire) [reply test/garbage.spec.js]: this shouldn't be removed

Comment (dignifiedquire) [reply test/garbage.spec.js]: we need to be able to set these values when testing

Comment (hugomrdias) [reply test/garbage.spec.js]: sure but these are tests and nothing is setting these env vars

how would you like this to be fixed ?

Comment (vmx) [reply test/garbage.spec.js]: Though it's neither document nor CI is using it. When is this test ever skipped (`NO_GARBAGE` is set)?

Comment (hugomrdias) [reply test/garbage.spec.js]: the test does nothing probably some left over from history

Comment (dignifiedquire) [reply test/garbage.spec.js]: I used this, and manually set it when I working on a fast iteration cycle as this is a pertty slow test. Please preserve the functionality that is there, allowing configuration of this test through setting env variables

**Comment by hugomrdias** [comment]
should be good to go and ci is passing https://travis-ci.org/github/dignifiedquire/borc/builds/666838072

**Review by vmx — APPROVED** [review]
@dignifiedquire can you do the merge and release stuff if you approve?

**Review by dignifiedquire — APPROVED** [review]

**Comment by hugomrdias** [comment]
@dignifiedquire can you release please ?

**Pull request merged**